ADA/ada.qmd

```diff
@@ -0,0 +1,75 @@
+---
+  title: "Analytical Data Access (ADA) and databricks"
+---
+  
+  <p class="text-muted">Guidance for analysts on how to interact with and use data stored in ADA using databricks</p>
+    
+---
+
+# About the ADA project
+
+---
+
+The [Analytical Data Access (ADA) service](https://analytical-data-access.azurewebsites.net/) has been created to support analysts, engineers and policy team members.
+
+It brings together:
+
+*	a searchable data catalogue - the Data Discovery Platform (DDP)
+*	a workbench of tools for analysing data - including Databricks
+*	a form for requesting access to data
+*	a support section - with walkthrough instructions
+*	a collection of data reports and dashboards
+*	a news and updates area.
+
+As long as you're connected to the network - it's available to anyone who works for the Department of Education.
+
+See these [Databricks support notebooks](https://adb-6882499576863257.17.azuredatabricks.net/browse/folders/3070168888967918?o=6882499576863257#) for more detailed reference guides into how we have setup the service here at DfE, including more information on workspaces, environments, notebooks, clusters, catalogs etc. 
+
+
+---
+
+# Plans for the future
+
+---
+
+The ambition is to enable access to all datasets across DfE and ESFA and retire the legacy applications. This will create common ways of working.
+
+---
+
+# Useful links
+
+---
+
+---
```

Comment:
Could remove this as we don't need two line breaks here if there's no text inbetween
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/c01dcb6b-3629-427b-85da-4a5305376a06)